create_infotexts.py

```diff
@@ -50,16 +50,16 @@ def load_keywords_mapping():
     :return: dictionary
     """
     kw_maps_url = "https://commons.wikimedia.org/wiki/Commons:Medelhavsmuseet/batchUploads/Cypern_keywords"
-    keywords = pd.read_html(kw_maps_url, attrs={"class": "wikitable sortable"}, header=0)
-    kw1 = keywords[0] # First is the actual column <Nyceklord>
-    
-    kw1 = kw1.set_index("Nyckelord")
-    kw1 = kw1[["Commons category", "wikidata"]] # Remove col "fredquency" for boolean filtering to work
-    
-    kw1[kw1 == "-"] = None  # Pandas "boolean indexing" kung-fu to replace all "-" with None
+    tables = pd.read_html(kw_maps_url, attrs={"class": "wikitable sortable"}, header=0)
+    keywords = tables[0] # First table is the actual column <Nyceklord>
```

Comment:
Formatting for comments should be `<code>  # <comment>` i.e. two spaces before and one after (also true for some other comments).

Could possibly also clarify the comment by changing it to `# First table is the one corresponding to <Nyckelord>` since column becomes a bit confusing when we are talking about tables (also fixed typo).